### ISSUE TITLE
Divine Purpose

### ISSUE THREAD

**Issue by Shiftt927** [Athissa/Tracker#237]
**Describe the issue you're having**:
Holy Power spending abilitys have 20% chance to give me proc which makes next holy power spender ability cost free
**Explain how you expect it work**:
It doesn't give proc.
**Steps to reproduce the problem**:

1. choose Divine Purpose talent
2. Hit with Teplars Verdict or Divine Storm
3. No proc

**Links to Wowhead, YouTube, etc**:

**Comment by AGrygor** [comment]
Justicar Vengeance makes it proc

**Comment by Shiftt927** [comment]
Yeah I noticed that while testing Justicar Vengeance, but what should I do because on that talent I'm using eye for an eye (for both pvp and pve) then I can't proc or ?

**Comment by GeneralissimeDarklinks** [comment]
je n'ai pas de soucis rencontré lors du proc du dessein Divin il proc bien sur mes trois sorts
